test/e2e/volume_provisioning.go

```diff
@@ -103,48 +103,46 @@ var _ = framework.KubeDescribe("Dynamic provisioning", func() {
-		framework.KubeDescribe("DynamicProvisioner", func() {
-			It("should create and delete persistent volumes [Slow] [Volume]", func() {
-				framework.SkipUnlessProviderIs("openstack", "gce", "aws", "gke", "vsphere")
-
-				By("creating a StorageClass")
-				class := newStorageClass("", "internal")
-				class, err := c.StorageV1beta1().StorageClasses().Create(class)
-				defer c.StorageV1beta1().StorageClasses().Delete(class.Name, nil)
-				Expect(err).NotTo(HaveOccurred())
-
-				By("creating a claim with a dynamic provisioning annotation")
-				claim := newClaim(ns)
-				claim.Spec.StorageClassName = &class.Name
-
-				defer func() {
-					c.Core().PersistentVolumeClaims(ns).Delete(claim.Name, nil)
-				}()
-				claim, err = c.Core().PersistentVolumeClaims(ns).Create(claim)
-				Expect(err).NotTo(HaveOccurred())
-
-				if framework.ProviderIs("vsphere") {
-					// vsphere provider does not allocate volumes in 1GiB chunks, so setting expected size
-					// equal to requestedSize
-					testDynamicProvisioning(c, claim, requestedSize)
-				} else {
-					// Expected size of the volume is 2GiB, because the other three supported cloud
-					// providers allocate volumes in 1GiB chunks.
-					testDynamicProvisioning(c, claim, "2Gi")
-				}
-			})
+	framework.KubeDescribe("DynamicProvisioner", func() {
+		It("should create and delete persistent volumes [Slow] [Volume]", func() {
+			framework.SkipUnlessProviderIs("openstack", "gce", "aws", "gke", "vsphere")
+
+			By("creating a StorageClass")
+			class := newStorageClass("", "internal")
+			class, err := c.StorageV1().StorageClasses().Create(class)
+			defer c.StorageV1().StorageClasses().Delete(class.Name, nil)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("creating a claim with a dynamic provisioning annotation")
+			claim := newClaim(ns)
+			claim.Spec.StorageClassName = &class.Name
+
+			defer func() {
+				c.Core().PersistentVolumeClaims(ns).Delete(claim.Name, nil)
+			}()
+			claim, err = c.Core().PersistentVolumeClaims(ns).Create(claim)
+			Expect(err).NotTo(HaveOccurred())
+
+			if framework.ProviderIs("vsphere") {
+				// vsphere provider does not allocate volumes in 1GiB chunks, so setting expected size
+				// equal to requestedSize
+				testDynamicProvisioning(c, claim, requestedSize)
+			} else {
+				// Expected size of the volume is 2GiB, because the other three supported cloud
+				// providers allocate volumes in 1GiB chunks.
+				testDynamicProvisioning(c, claim, "2Gi")
+			}
 		})
-	*/
+	})
+
 	framework.KubeDescribe("DynamicProvisioner Beta", func() {
 		It("should create and delete persistent volumes [Slow] [Volume]", func() {
 			framework.SkipUnlessProviderIs("openstack", "gce", "aws", "gke")
 
 			By("creating a StorageClass")
 			class := newStorageClass("", "beta")
-			_, err := c.StorageV1beta1().StorageClasses().Create(class)
-			defer c.StorageV1beta1().StorageClasses().Delete(class.Name, nil)
+			_, err := c.StorageV1().StorageClasses().Create(class)
```

Comment:
Good idea, I added a new commit with that change, a bit more lines of code were necessary to use v1beta1 API.